### ISSUE TITLE
Blizz Objective Tracker

### ISSUE THREAD

**Issue by joeymagz** [FailcoderAddons/supervillain-ui#78]
So, the Blizzard Objective Tracker displays in my Minimap all of a sudden and it won't go away. I can move it out of the way, but it interferes with other items. The box is MASSIVE and very annoying, if anyone knows a fix I would be really grateful.

![objectives](https://cloud.githubusercontent.com/assets/22799809/23622876/f3f20e2e-0265-11e7-9802-738bb5d1b9c1.PNG)

**Comment by finalsliver** [comment]
Do you have SVui_Quest tracker enabled? It's one of the addon modules

I seem to be able to replicate the issue by disabling the quest tracker so may be the issue. 

If the issue persists: did you use the Github or Curse/WoWinterface version of SVui,

**Comment by joeymagz** [comment]
Sorry for the delayed response. The QuestTracker is enabled and the issue still popped up. The problem started after I installed an addon called AngryKeystones and persisted after I deleted it.

I just deleted my old install including cache, downloaded the build from GitHub, installed fresh and the problem is gone. However, now I need to overwrite some of the files with the fixes I did manually...

**Comment by finalsliver** [comment]
Manual fixes?

If these are within SVui would you be able to create an issue with these? were currently in the process of tidying and fixing a few things and any work we can enhance the better & quicker we can bring SVui up-to-date. :D

**Comment by joeymagz** [comment]
BreakStuff and SVUI_Chat fixes posted... I had another one for mythic keystones, but it looks like someone with the username richy004 already posted an update with that one on his github... and his fix is cleaner than mine was.

**Comment by finalsliver** [comment]
Were currently set up on a Discord server as well, Richard is one of our new contributors. :)

You're welcome to join, chat flows between UI to other things. https://discord.gg/58AcatX

**Comment by joeymagz** [comment]
Found a fix for this. I came across some more addons that make an empty blizzard quest tracker show up, but this gets rid of it:

add this to SVUI_QuestTracker.lua line 488
> if (ObjectiveTrackerFrame:IsVisible() or ObjectiveTrackerFrame:IsShown()) then ObjectiveTrackerFrame:Hide(); end

This will hide the ObjectiveTrackerFrame no matter what.

Basically, any addons that effect quests or scenario's could potentially cause the issue. I've had the issue now with a few different addons for Mythic Plus tracking. (AngryKeystones and now GottaGoFast)
